applications_tests/gls_sharp_navier_stokes_3d/moving_stokes.prm

```diff
@@ -133,7 +133,8 @@ set time dependent =true
     end
     subsection bc 5
 	set id = 5
-        set type              = function
+        set type              = function weak
```

Comment:
Can you tell me why you made that change? Same question for the other .prm

Reply:
This case has Dirichlet boundary conditions everywhere. This means that there is no reference pressure. we were using the pressure reference point of the particle outside the particle to define a reference pressure. We cannot do this anymore (and it was a bad use of the pressure reference point). Instead, we now use the weak Dirichlet boundary conditions which still act as a pressure reference point.

Reply:
Seems like the weak dirichlet BC are getting more and more useful...